docs/modules/ROOT/pages/getting-started/configuration.adoc

```diff
@@ -0,0 +1,257 @@
+= Configuring KubeArchive
+
+This document explains how to configure KubeArchive so you can archive 
+resources and query them.
+
+== Prerequisites
+* KubeArchive installed and running in a cluster (xref:getting-started/installation.adoc[])
+* Know how to query the KubeArchive API (xref:getting-started/kubearchive-api.adoc[])
+
+== The KubeArchiveConfig resource
+
+To configure KubeArchive to archive or delete resources create a
```

Comment:
Even though we use `deleteWhen` as key for removing resources from etcd I think we should establish a difference between "delete a resource" (meaning, removing it forever) and "remove from the kubernetes cluster storage or etcd". We currently don't provide a feature for "deleting resources from KubeArchive database" but maybe we are in the future so I'd like to be more explicit about this.

If I'm not mistaken "delete" usually imply unrecoverable while "remove" isn't that extreme meaning "take away" or "set aside" so I would suggest we use "remove" in our docs to prevent confussion.

```suggestion
To configure KubeArchive to archive or remove resources from the cluster create a
```

Reply:
I tihnk we should either use "delete" or change to "removeWhen".

Reply:
And I actually vote to keep delete. We are in fact deleting it from the cluster. It no longer exists there.

> We currently don't provide a feature for "deleting resources from KubeArchive database" but maybe we are in the future so I'd like to be more explicit about this.

I view this as database work which is outside the scope of KubeArchive. Cleaning data from the database is not something I think KubeArchive should ever do.

Reply:
> I tihnk we should either use "delete" or change to "removeWhen".

I don't think that both things are related, the resource is being "deleted" from the cluster (in fact, the command is `kubectl delete`) but the resource isn't deleted, but removed if considered the big picture because it's kind of "moved" from etcd to an external place.

> I view this as database work which is outside the scope of KubeArchive. Cleaning data from the database is not something I think KubeArchive should ever do.

Even though if we don't consider having this feature in the scope of KubeArchive, I still find "deletion" a hard word considering that the resources are still available to query.

If you think that using the "delete" word isn't going to cause any missunderstanding, I'm ok with leaving it, but I thought that the "softness" of the "remove" word is more clear with the concept.

Reply:
I agree with keeping delete. I can add the "from the cluster" to make it more specific.

Reply:
> However, this is all just semantics which we can debate all day. :-)

Right, and that's why I'm open to keep the "delete" even if the "remove" sounds better for me, but it's ok, at the end we all mean the same idea.

> I can add the "from the cluster" to make it more specific.

This is fine.

>  You can't see them using kubectl or any other client that provides access to the cluster.

Be careful with this statement, as the cli provided by KubeArchive is provided as a `kubectl` plugin, meaning you can actually retrieve the resources with `kubectl` if the plugin is installed.

Feel free to close these threads @rh-hemartin :)

Reply:
> Be careful with this statement, as the cli provided by KubeArchive is provided as a kubectl plugin, meaning you can actually retrieve the resources with kubectl if the plugin is installed.

This is a great point. I forgot about that. But if I install the plugin then I would say I already know about KubeArchive.